### PR TITLE
Add resource manager packages compatible with the 2020-09-01-hybrid profile API versions

### PR DESCRIPTION
**Pull request by bbridges**
This PR adds new Track 2 resource manager packages which were copied and regenerated in order to be compatible with API versions in the 2020-09-01-hybrid profile. Minimal changes were made against the packages to get this to work against these API versions, and resource types which are found in the resource manager packages but not in the hybrid profile were kept to reduce the amount of required changes. These new modules were placed in a new `/sdk/resourcemanagerhybrid/` folder.

The group ID / artifact ID of these modules are the same as the existing Track 2 packages, and to reduce conflicts, they are not included in the root POM file. Instead, the versions are suffixed with `-hybrid` to tell them apart.

### CODE REVIEW

**Comment by bganapa** [comment]
Thanks @bbridges for the extra effort on making cleaner commits here!

@weidongxu-microsoft  could you please review the commits and help with build changes? We really appreciate your help and support here

**Comment by weidongxu-microsoft** [comment]
I've build it locally, code seems good.

Engineering part, one thing that need to be solved is the versioning integration at https://github.com/Azure/azure-sdk-for-java/blob/main/eng/versioning/version_client.txt#L198
We probably need help from engineering as we got same artifactId but different version.

Please locally run "mvn checkstyle:check" and "mvn spotbugs:check" to solve the error if possible. You seems not using Java11+ for code generation, hence the auto-formatter does not take effect. You might had to re-generate these RPs.

File: sdk/resourcemanager/api-specs.json [hunk 1]
```diff
@@ -24,12 +24,25 @@
     "args": "--payload-flattening-threshold=1 --tag=package-2021-02 --add-inner=AppServiceCertificate --name-for-ungrouped-operations=ResourceProvider",
     "note": "Add status code '200' to 'syncFunctionTriggers'"
   },
+  "appservice-hybrid": {
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/api-specs.json:27]: It is easier for you to have the api-specs.json in resourcemanagerhybrid folder?

Comment (bbridges) [reply sdk/resourcemanager/api-specs.json:27]: Yes I had placed it there for using gulp. I considered placing these in a new `api-specs.json` in the resourcemanagerhybrid folder as well and adding a command-line option to specify which api-spec file should be used for doing codegen as well but didn't implement it.

**Comment by weidongxu-microsoft** [comment]
version check script is `.\eng\versioning\pom_file_version_scanner.ps1`, and you can run it locally.

still checking on how this works.

**Review by weidongxu-microsoft — APPROVED** [review]

**Comment by weidongxu-microsoft** [comment]
Wes OKed with the changed on the CI. We will follow-up if anything unexpected happens.

**Pull request merged**